core-metastore/src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -30,6 +30,15 @@ public SchemaRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper obje
                 new SchemaRowMapper());
     }
 
+    public List<EventTypeSchema> listSchemas(final String name, final String version) {
```

Comment:
the method is not used, and there is a similar method

Reply:
removed, thanks escaped my notice :)